palantirnet/federated-search-demo
# PL-46: D7 domain examples

## JIRA Ticket(s)

- See ticket [PL-46: Add D7 Domain site to demo repo](https://palantir.atlassian.net/browse/PL-46).

## Description

Add Drupal 7 site with Domain example.

_This includes work from #12 since that deals with some of the same files._

_This is a rework of #11 with changes including those in #12._

## Testing instructions

- Run `phing build install-all` from inside the VM
- Go to http://d7-1.fs-demo.local
- Go to http://d7-2.fs-demo.local
- Go to http://d7-3.fs-demo.local
- Go to http://d7-1.fs-demo.local/search-app?search=ex to search for content
- Confirm that you can see content from the single D7 site and the D7 domain sites

## Relevant Screenshots/gifs:

![example](https://user-images.githubusercontent.com/397902/50698744-619eea00-100b-11e9-95cf-118f703788d8.jpg)

---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
